web/src/pages/oss.js

```diff
@@ -90,7 +102,10 @@ export default function Oss() {
         <div className="container">
           <h2>Repositories</h2>
           <div className="row is-multiline">
-            {repos.map((props, idx) => (
+            {repos.length < 1 && <div>
+              {/* TODO Add Spinner Here */}
```

Comment:
How would a loader or spinner be implemented? Or is there a better/easier solution?

Reply:
Not sure to be honest with docusaurus